### DISCUSSION TITLE
Error while building moose

### DISCUSSION THREAD

**Discussion by Ogugua99** [idaholab/moose#26266] [Q&A General]
I am trying to build moose on my Linux machine but i got the following error towards the end 
 {standard input}: Assembler messages:
{standard input}:425422: Warning: end of file not at end of a line; newline inserted
{standard input}:426148: Error: expecting operand after ','; got nothing
{standard input}: Error: open CFI at the end of file; missing .cfi_endproc directive
x86_64-conda-linux-gnu-c++: fatal error: Killed signal terminated program cc1plus
compilation terminated.
make: *** [/home/unokiwep/projects/moose/framework/build.mk:150: /home/unokiwep/projects/moose/test/build/unity_src/materials_Unity.x86_64-conda-linux-gnu.opt.lo] Error 1
make: *** Waiting for unfinished jobs....
(moose) unokiwep@Patrick:~/projects/moose/test$

**Comment by GiudGiud** [comment]
Hello

This looks like an OOM error with extra messages.
Can you please try `make -j1` ? This will use less memory

Guillaume

**Reply by jmeier** [reply]
I just had the same error message as @Ogugua99. I can confirm, that lowing the number of jobs (i.e. the number after the parameter -j) solved this problem as suggested by you @GiudGiud. For me "-j10" was fine.
